canmatrix/dbc.py

```diff
@@ -455,9 +455,9 @@ class FollowUps(object):
                                      dbcImportEncoding),
                                  receiver=receiver,
                                  multiplex=multiplex)
-                if not tempSig.is_little_endian:
+                #if not tempSig.is_little_endian:
```

Comment:
Can you please add a comment here basically copying the info from the PR? It'll help a future dev out a lot.